src/styles/App.css

```diff
@@ -0,0 +1,7 @@
+* {
+  color: white;
+}
+
+.weather-app {
+  margin: 1rem;
```

Comment:
A note about units here, its best to use rem for font sizes, em for things like margin and padding, and px for small things like border widths. This will be covered more in the styling lectures later on so dont worry about it too much right now, for reasons why have a look at [this article](https://dev.to/theodorusclarence/back-to-basic-should-we-use-rem-em-or-pixel-1hd0)